cirrus/google_cloud/manager.py

```diff
@@ -306,6 +306,60 @@ def get_project_organization(self):
             org = info["parent"]["id"]
         return org
 
+    def get_project_members(self):
```

Comment:
this should take an optional argument `project_id=None` and default to `self.project_id` if not provided